### PR TITLE
front2+3 paths

### PR DESCRIPTION
**Pull request by schen479**
Before issuing a pull request, please see the contributing page.

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/auton/Trajectories.java [hunk 1]
```diff
@@ -23,15 +24,55 @@ public Pose2d reflectY(Pose2d pose) {
     public class Far {
         // SPIKEMARK
         public Action MID_SPIKEMARK = drive.actionBuilder(drive.pose)
-
+                .lineToY(Positions.modifyPose(Positions.FAR.SPIKEMARK_MID, 0, Constants.ROBOT_LENGTH / 2).y)
+                .lineToY(Positions.modifyPose(Positions.FAR.SPIKEMARK_MID, 0, Constants.ROBOT_LENGTH / 2).y + 5)
```
[the remark this answers is not in the document]
Comment (schen479) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/auton/Trajectories.java:28]: cuz imma do my own thang

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/auton/Trajectories.java [hunk 2]
```diff
@@ -23,29 +24,75 @@ public Pose2d reflectY(Pose2d pose) {
...
+        .waitSeconds(1)
+                .setTangent(160)
+                .splineToLinearHeading(new Pose2d(Positions.modifyPose(Positions.GENERAL.BACKDROP_MID, -Constants.TILE_LENGTH,0).x, Positions.GENERAL.STACK_ONE.y, 0), Math.toRadians(180))
+                .strafeTo(new Vector2d(Positions.modifyPose(Positions.GENERAL.STACK_ONE, Constants.ROBOT_LENGTH/2, 0).x, Positions.GENERAL.STACK_ONE.y))
+                .lineToX(Positions.modifyPose(Positions.GENERAL.BACKDROP_RIGHT, -Constants.TILE_LENGTH, 0).x)
+                .splineToLinearHeading(Positions.vectorToPose(Positions.modifyPose(Positions.GENERAL.BACKDROP_MID, -Constants.ROBOT_LENGTH/2, 0), 0), Math.toRadians(0))
                 .build();
 
         public Action RIGHT_SPIKEMARK = drive.actionBuilder(drive.pose)
+                .lineToY(50)
```
Comment (Bi1ku) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/auton/Trajectories.java:46]: not right

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/auton/Trajectories.java [hunk 3]
```diff
@@ -23,29 +24,75 @@ public Pose2d reflectY(Pose2d pose) {
...
                 .build();
 
-        public Action RIGHT_BACKDROP = drive.actionBuilder(drive.pose)
-                .build();
 
-        public Action LEFT_BACKDROP = drive.actionBuilder(drive.pose)
-                .setTangent(0)
-                .splineToConstantHeading(Positions.modifyPose(Positions.GENERAL.BACKDROP_LEFT, -Constants.ROBOT_LENGTH / 2, 0), Math.toRadians(90))
+        public Action LEFT_SPIKEMARK = drive.actionBuilder(drive.pose)
+                .lineToY(50)
```
Comment (Bi1ku) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/auton/Trajectories.java:69]: not right, should be spikemark setup

**Review by Bi1ku — APPROVED** [review]

**Review by Bi1ku — APPROVED** [review]
approved and merge given you make the necessary changes

**Pull request merged**